components/composited/ProjectCard/ProjectCard.tsx

```diff
@@ -0,0 +1,86 @@
+import { FormEventHandler, Fragment, useCallback, useState } from 'react';
+import { Badge } from '../../generic/Badge/Badge';
+import { Heading } from '../../generic/Heading/Heading';
+import { ExpandLessIcon } from '../../generic/Icons/ExpandLessIcon';
+import { ExpandMoreIcon } from '../../generic/Icons/ExpandMoreIcon';
+import { Text } from '../../generic/Text/Text';
+import { TextButton } from '../../generic/TextButton/TextButton';
+import { VoteButton } from '../../generic/VoteButton/VoteButton';
+import { ProjectInfoList } from './ProjectInfoList';
+
+type ProjectCardFields = {
+  title: string;
+  description: string;
+  estimatedCost: string;
+  estimatedRealizationDate: string;
+  place: string;
+  voted: boolean;
```

Comment:
Add object, such as `details`. It might change in the future, so don't make it explicit here.

Reply:
Wdym by not explicit here?

Reply:
I mean that you pass an object `details` as a prop that has all those fields.